files/grest/rpc/cached_tables/pool_history_cache.sql

```diff
@@ -30,6 +28,16 @@ declare
   _curr_epoch bigint;
   _latest_epoch_no_in_cache bigint;
 begin
+   IF (
+    SELECT
+      COUNT(pid) > 1
+    FROM
+      pg_stat_activity
+    WHERE
+      query != '<IDLE>' AND query ILIKE '%GREST.pool_history_cache_update%') THEN
```

Comment:
Can we change this line to:
`state = 'active' AND query ILIKE '%GREST.pool_history_cache_update%'`